src/index.js

```diff
@@ -0,0 +1,159 @@
+const state = {
+  temp: 45,
+  city: 'Seattle',
+};
+
+//converting Kelvin to Fahrenheit
+const convertKelvinToF = (temp) => ((temp - 273.15) * 9) / 5 + 32;
+
+//Creating addTemp and minus Temp events
+const addTemp = (event) => {
+  state.temp = Math.round(state.temp + 1);
+  const tempControl = document.querySelector('#currentTemp');
+  tempControl.textContent = state.temp;
+  changeTempNumColor();
+};
+
+const minusTemp = (event) => {
+  state.temp = Math.round(state.temp - 1);
+  const tempControl = document.querySelector('#currentTemp');
+  tempControl.textContent = state.temp;
+  changeTempNumColor();
+};
```

Comment:
I recommend refactoring these two functions to DRY the code - one possibility would be to create a helper function that  does this same work and uses a parameter to determine how much to change the temperature.